### ISSUE TITLE
I think there is error in focal loss

### ISSUE THREAD

**Issue by suminlee94** [andrijdavid/FocalLoss#2]
```
 logpt = - F.binary_cross_entropy_with_logits(input, target, pos_weight=weight, reduction=self.reduction)
pt = torch.exp(logpt)

 # compute the loss
focal_loss = -( (1-pt)**self.gamma ) * logpt
```
'logpt' is actually -log(p_t).
so....pt is not same as log(sigmoid(input)).
